promql/promqltest/testdata/functions.test

```diff
@@ -115,11 +109,10 @@ load 5m
 
 # Counter resets at in the middle of range are handled correctly by rate().
 eval instant at 50m rate(testcounter_reset_middle[50m])
-	{} 0.03
+	{} 0.02962962962962963
 
 # Counter resets at end of range are ignored by rate().
 eval instant at 50m rate(testcounter_reset_end[5m])
-	{} 0
```

Comment:
The point of this test is to yield 0, not an empty result, so you need to extend the range here. 6m should work.